### PR TITLE
Added location to gamelift_fleet

### PR DESCRIPTION
**Pull request by 20chan**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request

Output from acceptance testing:

```
$ make testacc TESTARGS='-run=TestAccAWSGameliftFleet'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./aws -v -count 1 -parallel 20 -run=TestAccAWSGameliftFleet -timeout 180m
=== RUN   TestAccAWSGameliftFleet_basic
=== PAUSE TestAccAWSGameliftFleet_basic
=== RUN   TestAccAWSGameliftFleet_tags
=== PAUSE TestAccAWSGameliftFleet_tags
=== RUN   TestAccAWSGameliftFleet_allFields
=== PAUSE TestAccAWSGameliftFleet_allFields
=== RUN   TestAccAWSGameliftFleet_disappears
=== PAUSE TestAccAWSGameliftFleet_disappears
=== CONT  TestAccAWSGameliftFleet_basic
=== CONT  TestAccAWSGameliftFleet_disappears
=== CONT  TestAccAWSGameliftFleet_allFields
=== CONT  TestAccAWSGameliftFleet_tags
--- PASS: TestAccAWSGameliftFleet_basic (1804.84s)
--- PASS: TestAccAWSGameliftFleet_disappears (1889.02s)
--- PASS: TestAccAWSGameliftFleet_tags (1912.98s)
--- PASS: TestAccAWSGameliftFleet_allFields (2027.51s)
PASS
ok      github.com/terraform-providers/terraform-provider-aws/aws       2027.585s
```

### CODE REVIEW

File: aws/resource_aws_gamelift_fleet.go [hunk 1]
```diff
@@ -70,6 +70,19 @@ func resourceAwsGameliftFleet() *schema.Resource {
 				Optional:     true,
 				ValidateFunc: validation.StringLenBetween(1, 1024),
 			},
+			"locations": {
```
Comment (ewbankkit) [inline aws/resource_aws_gamelift_fleet.go:73]: I suggest renaming this argument to `location_configuration` to match the underlying AWS API.

**Comment by 20chan** [comment]
@ewbankkit Sorry for asking multiple times, but can you tell me how's reviewing going on?

**Comment by ewbankkit** [comment]
@20chan Thanks for continuing to work on this. Everything is looking good.
Could you please add an [acceptance test](https://github.com/hashicorp/terraform-provider-aws/blob/main/docs/contributing/running-and-writing-acceptance-tests.md#writing-an-acceptance-test) for the new argument.

**Comment by 20chan** [comment]
@ewbankkit  Thanks. Tests are added and here's output of tests:

```shell
$ make testacc TESTARGS='-run=TestAccAWSGameliftFleet'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./aws -v -count 1 -parallel 20 -run=TestAccAWSGameliftFleet -timeout 180m
=== RUN   TestAccAWSGameliftFleet_basic
=== PAUSE TestAccAWSGameliftFleet_basic
=== RUN   TestAccAWSGameliftFleet_tags
=== PAUSE TestAccAWSGameliftFleet_tags
=== RUN   TestAccAWSGameliftFleet_allFields
=== PAUSE TestAccAWSGameliftFleet_allFields
=== RUN   TestAccAWSGameliftFleet_disappears
=== PAUSE TestAccAWSGameliftFleet_disappears
=== CONT  TestAccAWSGameliftFleet_basic
=== CONT  TestAccAWSGameliftFleet_disappears
=== CONT  TestAccAWSGameliftFleet_tags
=== CONT  TestAccAWSGameliftFleet_allFields
--- PASS: TestAccAWSGameliftFleet_basic (2091.79s)
--- PASS: TestAccAWSGameliftFleet_disappears (2106.49s)
--- PASS: TestAccAWSGameliftFleet_tags (2181.19s)
--- PASS: TestAccAWSGameliftFleet_allFields (2277.32s)
PASS
ok      github.com/terraform-providers/terraform-provider-aws/aws       2277.477s
```

**Comment by 20chan** [comment]
@ewbankkit I'm really sorry for keep pushing you on this but could you check again?

**Comment by NicholasLebrun** [comment]
Any update on this? We would love to use @20chan 's work. It seems like tests have passed but it hasn't been merged in. I'm sorry for not adding anything relevant to the conversation, just wondering if/when we'll have this feature available.

**Pull request closed without merging**